### PR TITLE
Make checkboxes horizontal on contact form for eks enterprise

### PR DESCRIPTION
**Pull request by yorinasub17**
Addresses https://github.com/gruntwork-io/gruntwork-io.github.io/pull/585#issuecomment-900182468

### CODE REVIEW

File: pages/landing/enterprise-eks/contact/_form.html [hunk 1]
```diff
@@ -39,42 +39,35 @@ <h1 style="text-align: center;">
...
           <div class="col-xs-12 col-md-4">
-            <label>Which features are you interested in?</label>
-          </div>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="row">
-          <div class="col-xs-12 col-md-4">
-            <label>Service Mesh</label>
+            <label style="padding-top: 0;">Service Mesh</label>
```
Comment (brikis98) [inline pages/landing/enterprise-eks/contact/_form.html:42]: NIT: all the `<label>` elements in the left column need the `padding-top` fix; it looks like you've added it here, but "Monitoring", "App languages / frameworks", and "Security / Compliance" still need it.

File: pages/landing/enterprise-eks/contact/_form.html [hunk 2]
```diff
@@ -39,42 +39,35 @@ <h1 style="text-align: center;">
...
           </div>
           <div class="col-xs-12 col-md-8">
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="istio" value="Istio" %}
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="consul-connect" value="Consul Connect" %}
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="aws-app-mesh" value="AWS App Mesh" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="istio" value="Istio" width="145px" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="consul-connect" value="Consul Connect" width="145px" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="aws-app-mesh" value="AWS App Mesh" width="160px" %}
```
Comment (brikis98) [inline pages/landing/enterprise-eks/contact/_form.html:47]: NIT: swap the order of Consul Connect and AWS App Mesh; the spacing will look a tiny bit better.

**Review by brikis98 — APPROVED** [review]
LGTM! Just two tiny spacing tweaks, and this is good to go.

File: pages/landing/enterprise-eks/contact/_form.html (hunk 1 above)
Comment (yorinasub17) [reply pages/landing/enterprise-eks/contact/_form.html:42]: Done: 7911607

File: pages/landing/enterprise-eks/contact/_form.html (hunk 2 above)
Comment (yorinasub17) [reply pages/landing/enterprise-eks/contact/_form.html]: Done: 7911607

**Review by brikis98 — APPROVED** [review]
LGTM, thanks!

**Comment by yorinasub17** [comment]
Thanks for review! Merging now.

**Pull request merged**